staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go

```diff
@@ -44,8 +44,10 @@ var ErrAPIServiceNotFound = errors.New("resource not found")
 // known specs including the http etag.
 type SpecAggregator interface {
 	AddUpdateAPIService(apiService *v1.APIService, handler http.Handler) error
-	UpdateAPIServiceSpec(apiServiceName string) error
-	RemoveAPIService(apiServiceName string) error
+	// UpdateAPIServiceSpec updates the APIService. It returns a bool which indicates whether the APIService exists and
+	// the actual error.
+	UpdateAPIServiceSpec(apiServiceName string) (bool, error)
```

Comment:
this is not what I meant. If you make "not found" explicit, it is not an error anymore. So we would return `false, nil`.